packages/react-components/react-swatch-picker-preview/src/components/EmptySwatch/useEmptySwatch.ts

```diff
@@ -22,6 +22,7 @@ export const useEmptySwatch_unstable = (
   const isGrid = useSwatchPickerContextValue_unstable(ctx => ctx.isGrid);
 
   const role = isGrid ? 'gridcell' : 'radio';
+  const a11yProps = isGrid ? {} : { 'aria-checked': false };
```

Comment:
this feels wrong, if it should be empty lets use `null`

Reply:
fixed